Update: List transformation README (#934)
Esri/arcgis-runtime-samples-ios: arcgis-ios-sdk-samples/Geometry/List transformations/README.md

Some transformations aren't available until transformation data is provided.

> This sample uses an `AGSGeographicTransformation` object, which extends the `AGSDatumTransformation` class. As of 100.9, *ArcGIS Runtime* also includes an `AGSHorizontalVerticalTransformation` class, which also extends `AGSDatumTransformation`. The `AGSHorizontalVerticalTransformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.

Reviewer: The verbiage "extending a class" has a different meaning in Swift.
```suggestion
This sample uses `AGSGeographicTransformation`, a subclass of `AGSDatumTransformation`. As of 100.9, *ArcGIS Runtime* also includes `AGSHorizontalVerticalTransformation`, another subclass of `AGSDatumTransformation`. The `AGSHorizontalVerticalTransformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
```

2. In the Dashboard page, click '[Download APIs and SDKs](https://developers.arcgis.com/downloads/data)' and go to the `Supplemental ArcGIS Runtime Data` tab.
3. Click the download button next to `Projection Engine Data` to download projection engine data to your computer.
4. Unzip the downloaded data on your computer.

> 5. Copy the `PEDataRuntime` folder to your application's Documents folder.?

Author: ```suggestion
5. Copy the `PEDataRuntime` folder to your application's Documents folder.
```
